x-pack/filebeat/module/salesforce/setupaudittrail/config/setupaudittrail.yml

```
@@ -0,0 +1,40 @@
+{{ if eq .input "salesforce" }}
+
+id: setupaudittrail
+type: salesforce
+enabled: true
+{{ if .api_version }}
+version: {{ .api_version }}
+{{ end }}
+auth.oauth2: {{ .authentication | tojson }}
+url: {{ .url }}
+event_monitoring_method:
+  object:
+    enabled: {{ if .real_time }}true{{ else }}false{{ end }}
+    interval: {{ .real_time_interval }}
+    query:
+      default: SELECT FIELDS(STANDARD) FROM SetupAuditTrail{{ if .initial_interval }} WHERE CreatedDate > [[ (formatTime (now.Add (parseDuration "-{{ .initial_interval }}")) "2006-01-02T15:04:05+0700") ]]{{ end }} ORDER BY CreatedDate ASC NULLS FIRST

```

```suggestion
      default: SELECT FIELDS(STANDARD) FROM SetupAuditTrail{{ if .initial_interval }} WHERE CreatedDate > [[ (formatTime (now.Add (parseDuration "-{{ .initial_interval }}")) "2006-01-02T15:04:05.000Z0700") ]]{{ end }} ORDER BY CreatedDate ASC NULLS FIRST

```

I think we should use this instead. This is as per the format received in the original logs 
```
 {
      "attributes": {
        "type": "SetupAuditTrail",
        "url": "/services/data/v54.0/sobjects/SetupAuditTrail/0Ym5j00003ZZWydCAH"
      },
      "Id": "0Ym5j00003ZZWydCAH",
      "Action": "PermSetEntityPermChanged",
      "Section": "Manage Users",
      "CreatedDate": "2024-04-09T12:12:23.000+0000",
      "CreatedById": "0055j00000AT6I1AAL",
      "Display": "Changed permission set Buyer: Contact Point Address object permissions were changed from No Access to Read, Create, Edit, Delete",
      "DelegateUser": null,
      "ResponsibleNamespacePrefix": null,
      "CreatedByContext": null,
      "CreatedByIssuer": null
    }